frontend/frontend/manager.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime, timedelta
+from typing import List, Dict, Any
+from pydantic import BaseModel
+
+class Answer(BaseModel):
```

Comment:
There are two definitions of `Answer`. It'd be nice to see some tests around that.

Reply:
I've got a weak test around this... I make sure both copies of `Answer` can be instantiated with one answer from a user.